src/runtime/profiling.cc

```diff
@@ -887,6 +887,8 @@ PackedFunc WrapTimeEvaluator(PackedFunc pf, Device dev, int number, int repeat,
         }
         t->Stop();
         int64_t t_nanos = t->SyncAndGetElapsedNanos();
+        if (t_nanos == 0) absolute_zero_times++;
+        if (absolute_zero_times >= max_repeat_num) break;
```

Comment:
It seems to me that in the case of repeating several zero measurements, we could try to change the "number" using a different formula: for special case when duration_ms is 0 and absolute_zero_times > 1. Otherwise I don't see the sense of the default value of 100 for "max_repeat_num" argument.

Reply:
It doesn't make sense. Because for operations such as `__nop` or `__copy` the `t_nanos` will be always zero. And if we will increase the `numbers` then we won't exit from the infinite loop. What about default value of 100. I used 100 because it is not too much and not too little :) If the user needs to change it, he can do it any time.